src/main/java/Duke.java

```diff
@@ -1,10 +1,54 @@
+import java.io.IOException;
+
+/**
+ * Driver class of Duke chatbot, also saves on startup and loads on exit.
+ */
```

Comment:
If you like, comments for members (not methods) can be on a single line:

/** Driver class of Duke chatbot, also saves on startup and loads on exit. */

Reply:
Looks much cleaner, thanks.